python/pyproject.toml

```diff
@@ -141,6 +141,7 @@ embeddings = [
 
 [tool.poetry.group.dev.dependencies]
 types-urllib3 = "^1.26.25.5"
+jupyter = "^1.0.0"
```

Comment:
This is ok, but do we need this?

Reply:
How else do you run the example notebooks?

Reply:
I configure my system to never allow global pip dependencies FYI, so maybe you just have a single jupyter instance on your system that you use for everything? It seems like its part of the dev process here to use notebooks so it seemed right to add it to the dev dependencies instead of making this one package an exception.

Reply:
There are many ways to run notebooks, whylogs doesn't specifically require jupyter, .ipynb is an open file format.

whylogs already depends on `ipykernel` and `papermill` as a dev dependencies, which lets us run notebooks in automated tests, and have left it up to people how they want to run the examples. We do lean on colab as a convenient way for users browsing our github repo to click a button to run them. We could add this as convenience for contributors but we run these in colab or an IDE directly while developing.